it/controllers/PenaltiesFrontendControllerISpec.scala

```diff
@@ -348,9 +348,6 @@ class PenaltiesFrontendControllerISpec extends IntegrationSpecCommonBase with ET
           |     "PoCAchievementDate": "2022-01-01"
           |   },
           |   "details": []
-          | },
-          | "latePaymentPenalty": {
```

Comment:
I know this is worth removing but how did you notice it? By having this does the tests fail?

Reply:
Yes the test failed due to an empty LPP list being added to summary model status returned etc still correct (204) so didn't think it warranted making code change

Reply:
Just thinking if this situation is possible?